Makefile

```diff
@@ -19,7 +19,7 @@ all: subdirs
 
 clean:
 	rm -f *.pyc *.pyo *~ *.bak
-	rm -f $(BUILDDIR)/{SOURCES,SRPMS,RPMS}/*
+	rm -f $(BUILDDIR)/SOURCES/* $(BUILDDIR)/SRPMS/* $(BUILDDIR)/RPMS}/*
```

Comment:
There's a typo: the brace at the end should also be removed.

Reply:
Thanks for the hint. The typo is now fixed.